scaffold-module-config.yaml

```diff
@@ -0,0 +1,19 @@
+name: "kyma-project.io/module/mymodule" # required, the name of the Module
```

Comment:
How is this file just created in this PR? 😄

Reply:
Whoops. Also the other one already in main should not have been committed 😆 
Removed both. 👍